questionnaire/questionnaire-service.js

```diff
@@ -215,7 +215,6 @@ function createQuestionnaireService({
             // if the caseReferenceNumber exists, then the submission is COMPLETE.
             if (caseReferenceNumber) {
                 await updateQuestionnaireSubmissionStatus(questionnaireId, 'COMPLETED');
```

Comment:
Should this not treated the same as sending a notification? e.g. they'll get a notification eventually, however their application will still show "IN_PROGRESS"

Reply:
If you are basically asking: "should we move the updating of the submission_status in to the `system answers` route handler?" then I did think about that yeah. Maybe it should be that way!